todo_list_project/tasks/tests.py

```
@@ -67,12 +89,39 @@ def test_task_toggle_complete_view(self):
+    def test_task_list_ordering(self):
+        Task.objects.all().delete()  # Limpiar todas las tareas existentes
+        Task.objects.create(title='Tarea A', user=self.user)
+        Task.objects.create(title='Tarea B', user=self.user)
+        response = self.client.get(reverse('task_list'))
+        tasks = list(response.context['tasks'])
+        self.assertEqual(tasks[0].title, 'Tarea A')
+        self.assertEqual(tasks[1].title, 'Tarea B')
+
+    def test_task_list_chronological_order(self):
+        Task.objects.all().delete()  # Limpiar todas las tareas existentes
+        task1 = Task.objects.create(title='Tarea 1', user=self.user)
```

Ruff

Local variable `task1` is assigned to but never used

Remove assignment to unused variable `task1`

(F841)